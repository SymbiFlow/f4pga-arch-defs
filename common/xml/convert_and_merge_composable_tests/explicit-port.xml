<?xml version="1.0"?>
<!-- Convert
   * <port name=XXX>                         to XXX
   * <port name=XXX bit=Y>                   to XXX[Y]
   * <port name=XXX bit-msb=M bit-lsb=L>     to XXX[M:L]
  -->
<xml>
  <port-value-test><port name="port-name" /></port-value-test>
  <port-value-test><port name="port-name" bit="8" /></port-value-test>
  <port-value-test><port name="port-name" bit-msb="8" bit-lsb="4" /></port-value-test>
</xml>
