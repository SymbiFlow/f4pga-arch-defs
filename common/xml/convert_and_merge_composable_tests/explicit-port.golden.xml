<?xml version="1.0"?>
<xml>
  <port-value-test o="port-name"/>
  <port-value-test o="port-name[8]"/>
  <port-value-test o="port-name[8:4]"/>
</xml>
